language-server/modules/langserver-core/src/test/resources/completion/query_expression/source/query_expr_ctx_limit_clause_source4.bal

```diff
@@ -0,0 +1,16 @@
+import ballerina/module1;
+
```

Comment:
Shall we include a `int`  constant and a function which returns int as well to ensure they are sorted properly as well

Reply:
+1

Reply:
+1

Reply:
Will add it.